critiquebrainz/frontend/external/musicbrainz_db/tests/artist_test.py

```diff
@@ -0,0 +1,46 @@
+from unittest import TestCase
+from unittest.mock import MagicMock
+from critiquebrainz.frontend.external.musicbrainz_db.test_data import artist_linkin_park, artist_jay_z
+from critiquebrainz.frontend.external.musicbrainz_db import artist as mb_artist
+from critiquebrainz.frontend.external.musicbrainz_db.tests import setup_cache
+
+
+class ArtistTestCase(TestCase):
+
+
```

Comment:
Should be just one blank line here. https://www.python.org/dev/peps/pep-0008/#blank-lines